gems/bundler/CVE-2013-0334.yml

```diff
@@ -2,11 +2,12 @@
 gem: bundler
 cve: 2013-0334
 osvdb: 110004
+ghsa: 49jx-9cmc-xjxm
 url: https://nvd.nist.gov/vuln/detail/CVE-2013-0334
-title: "CVE-2013-0334 rubygem-bundler: 'bundle install' may install a gem from a source\
-  \ other than expected"
+title: 'CVE-2013-0334 rubygem-bundler: ''bundle install'' may install a gem from a source other than expected'
```

Comment:
Maybe we should use double-quotes if the string contains single-quotes? Your call.